specs/plugin-extensibility.md

```diff
@@ -120,7 +120,7 @@ Plugin config can be also set/overriden during signing with the `notation sign`
 
 ### Plugin contract
 
-* Notation will invoke the plugin executable for each command (e.g. sign, verify), pass inputs through `stdin` and get output through `stdout` and `stderr`.
+* Notation will invoke the plugin executable for each command (e.g. sign, verify), pass inputs through `stdin` and get output through `stdout` and `stderr`. The output size is limited to less than 64 MiB to avoid potential plugin unexpected behavior that triggers out-of-memory issues.
```

Comment:
```suggestion
* Notation will invoke the plugin executable for each command (e.g. sign, verify), pass inputs through `stdin` and get output through `stdout` and `stderr`. The output size is limited to less than 64 MiB per channel to avoid potential plugin unexpected behavior that triggers out-of-memory issues.
```

Reply:
Updated.